src/pages/index.js

```diff
@@ -24,10 +35,40 @@ newPopupImage.setEventListeners();
 const cardPopup = new PopupWithForm("#addModal", handleCardFormSubmit);
 cardPopup.setEventListeners();
 
-const newCardList = new Section(
-  { items: constants.initialCards, renderer: createCard },
-  ".cards__list"
+const avatarPopup = new PopupWithForm(
+  "#editAvatarModal",
+  handleAvatarFormSubmit
 );
+avatarPopup.setEventListeners();
+
+const popupAffirm = new PopupWithConfirm("#deleteCardModal");
+popupAffirm.setEventListeners();
+
+let newCardList;
+let card;
```

Comment:
`let card;` is not needed here because you shouldn't reassign the variable for each card you create. It can be misleading

Reply:
You haven't corrected something https://github.com/GodisNanu/se_project_aroundtheus/pull/1#discussion_r1591348606